ImperatorToCK3/ImperatorToCK3.vcxproj

```diff
@@ -136,15 +136,17 @@
       <AdditionalDependencies>Resources\rakaly.dll.lib;CORE_RL_Magick++_.lib;CORE_RL_MagickCore_.lib;CORE_RL_MagickWand_.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
     <PostBuildEvent>
-      <Command>xcopy "$(SolutionDir)ImageMagick\dll" "$(OutDir)\ImperatorToCK3" /Y /E /I
+      <Command>xcopy "$(SolutionDir)ImageMagick\dll-windows-required" "$(OutDir)\ImperatorToCK3" /Y /E /I
```

Comment:
this is not the way. Either have it as a standalone bat file, or pull in all the files you need to copy into the project itself, right click them and select "Copy file", and then in options select destination. This is unmaintainable from outside.

Reply:
I'd need a second bat file for Debug config just to have all "Release" occurences replaced with "Debug". I will use Copy File.

Reply:
Done, got rid of Copy_Files.bat and made all the stuff Copy files.